howto/docker.rst

```diff
@@ -28,12 +28,15 @@ Docker image for Gauge
     RUN apt-get -q -y install default-jdk
     RUN apt-get install apt-transport-https -y
 
-    #Install gauge
+    # Install gauge
```

Comment:
This section violates docker's best practices :-(

This is not related to your PR, but since you are already here can you change multiple `RUN` command to a single multi-line one? https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run